content/en/account_management/rbac/permissions.md

```diff
@@ -0,0 +1,372 @@
+        -H "Content-Type: application/json" \
+        -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
+        -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>" \
+        -d '{
+                "data": {
+                    "type": "permissions",
+                    "id": <PERMISSION_UUID>,
+                    "scope": {
+                        "pipelines": [
+                            "abcd-1234",
+                            "bcde-2345"
+                        ]
+                    }
+                }
+            }'
+```
+
+
+[1]: /api/#roles
+[2]: /api/#get-permissions
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_processors
+
+Grants a role the ability to create or modify the processors within a processing pipeline.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+**Global access**:
+
+Go to your [Datadog Roles page][1] and select the checkbox `write` as below for the wanted role:
+
+{{< img src="account_management/rbac/logs_write_processors_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+**Subset of Pipelines**:
+
+1. Remove the global permission on the role.
+2. This permission can be granted to a role in [the Processing Pipelines page of the Datadog app][2] by editing a processing pipeline and adding a role to the "Grant editing Processors of this index to" field (screenshot below).
+
+{{< img src="account_management/rbac/logs_write_processors.png" alt="Grant write access for processors to specific roles"  style="width:75%;" >}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+[2]: https://app.datadoghq.com/logs/pipelines
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_archives
+
+Grants the ability to create or modify log archives.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_write_archives_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_public_config_api
+
+Grants the ability to create or modify log configuration through the Datadog API.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_public_config_api_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Data
```

Comment:
this header is pretty deeply nested and comes after a very long section. normally `Configuration` and `Data` would be great headers under `Advanced Permissions > Log Management`, but once i got to "Data" i was like "where am i? what does this mean?"

So maybe in this case we should make it a bit more explicit? `Configuration Access` and `Data Access`?